handlers/handlers.go

```diff
@@ -21,16 +22,17 @@ func setStatusCode(req *http.Request, w http.ResponseWriter, err error) {
 }
 
 // Bulletin handles bulletin requests
-func Bulletin(cfg config.Config, rc RenderClient, zc ZebedeeClient, ac ArticlesApiClient) http.HandlerFunc {
+func SixteensBulletin(cfg config.Config, rc RenderClient, zc ZebedeeClient, ac ArticlesApiClient) http.HandlerFunc {
 	return dphandlers.ControllerHandler(func(w http.ResponseWriter, r *http.Request, lang, collectionID, accessToken string) {
-		bulletin(w, r, accessToken, collectionID, lang, rc, zc, ac, cfg)
+		sixteensBulletin(w, r, accessToken, collectionID, lang, rc, zc, ac, cfg)
 	})
 }
 
-func bulletin(w http.ResponseWriter, req *http.Request, userAccessToken, collectionID, lang string, rc RenderClient, zc ZebedeeClient, ac ArticlesApiClient, cfg config.Config) {
+func sixteensBulletin(w http.ResponseWriter, req *http.Request, userAccessToken, collectionID, lang string, rc RenderClient, zc ZebedeeClient, ac ArticlesApiClient, cfg config.Config) {
 	ctx := req.Context()
-
-	bulletin, err := ac.GetLegacyBulletin(ctx, userAccessToken, collectionID, lang, req.URL.EscapedPath())
+	muxVars := mux.Vars(req)
+	uri := muxVars["uri"]
```

Comment:
I think this will not contain the leading `/` which we expect to be part of the uri?